src/main/java/org/alfresco/rest/model/RestRuleModelsCollection.java

```diff
@@ -0,0 +1,8 @@
+package org.alfresco.rest.model;
+
+import org.alfresco.rest.core.RestModels;
+
+public class RestRuleModelsCollection extends RestModels<RestRuleModel, RestRuleModelsCollection>
```

Comment:
I don't understand what this class is. It doesn't seem to be autogenerated and it doesn't seem to be used anywhere.

Reply:
It wraps entries in a list with pagination - it is somehow generic way of wrapping entries

Some of other paging objects from yaml are manually created inside TAS project, see RestActionDefinitionModelsCollection and the class it extends